app/admin/approval_delegate.rb

```diff
@@ -0,0 +1,34 @@
+ActiveAdmin.register ApprovalDelegate do
+  actions :index, :new, :show, :create, :destroy
+
+  permit_params :assigner_id, :assignee_id
+
+  form do |f|
+    f.inputs 'Delegate' do
+      f.input :assigner, as: :select, collection: User.all.map { |user| [ user.email_address, user.id] }
```

Comment:
I had to do this in order to get email address to show up rather than `user.full_name` -- any ideas on why that was the default behavior? Only thing I could find via googling was that ActiveAdmin uses `to_s` but we haven't monkeypatched that AFAIK